deployer/pipeline_deployer.py

```diff
@@ -106,6 +109,21 @@ def _check_experiment_name(self, experiment_name: Optional[str] = None) -> str:
 
         return experiment_name
 
+    def _check_pipeline_display_name(self, tag: Optional[str] = None) -> None:
+        now_str = datetime.now().strftime("%Y%m%d%H%M%S")
+
+        if self.pipeline_display_name is None:
+            self.pipeline_display_name = f"{self.pipeline_name}"
+            logger.info(f"display_name not provided, using {self.pipeline_name} as base")
+        if tag:
+            self.pipeline_display_name += f"-{tag}"
+        self.pipeline_display_name += f"-{now_str}"
+
+        self.pipeline_display_name = self.pipeline_display_name.replace("_", "-")
+        self.pipeline_display_name = self.pipeline_display_name[:127]
+
+        logger.info(f"Pipeline_display_name is: {self.pipeline_display_name}")
```

Comment:
maybe debug level is more accurate ? + isn't this already in the compile / run logs?

Reply:
- ok can change to debug
- I will check if the UI display name is already in the logs, I thought it was not